### ISSUE TITLE
Vue3 "script setup" mode

### ISSUE THREAD

**Issue by tarkhil** [AdamNiederer/vue-mode#117]
Is it possible to make syntax working in script setup sections?

**Comment by eric-p-hutchins** [comment]
As a workaround I've found that it works if you use:

```vue
<script setup="setup" lang="ts">
</script>
```

instead of

```vue
<script setup lang="ts">
</script>
```

but I agree it would be nice if it would act the same even using "setup" with no value.

**Comment by eric-p-hutchins** [comment]
Oh there's a pull request for it: https://github.com/AdamNiederer/vue-mode/pull/115

**Comment by dherbst** [comment]
This was fixed by PRs #115 and #123.  Please open a new issue if you encounter any problems.
